### ISSUE TITLE
question about codes

### ISSUE THREAD

**Issue by 1187697147** [facebookresearch/TimeSformer#5]
Hi, 
I met one problem about time embedding in your code:
```
cls_tokens = x[:B, 0, :].unsqueeze(1)
```
Why previous B tokens can represent the cls_token?
Looing for your reply!

**Comment by Hanqer** [comment]
@1187697147  In my view, each sample in the first dim of 'x' are same, because they are all expanded by cls token. So x[:B] is a simplified code style for the convenience  of 'torch.cat' without expand again.

**Comment by 1187697147** [comment]
Hi, @Hanqer 
Thank you for your reply! 
I get what you mean, this helps me understand time_embedding better!
